ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsUserManagement.scala

```diff
@@ -29,31 +30,93 @@ private[backend] trait StorageBackendTestsUserManagement
 
   private def tested = backend.userManagement
 
+  it should "handle created_at and granted_at attributes correctly" in {
+    val user = newUniqueUser()
+    val internalId = executeSql(tested.createUser(user, createdAt = 123))
```

Comment:
could we define it once only and use in both test and assertion?

Reply:
Ok, extracted `zeroMicros` val for that although in this particular test case I'm not using it